scripts/version.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-echo "Please summarize this release in one sentence"
-
-read note
-
+sed -E "s/MARKETING_VERSION = [0-9.]+/MARKETING_VERSION = $npm_package_version/" safari/Shared.xcconfig | tee safari/Shared.xcconfig
 dot-json assets/manifest.json version $npm_package_version
 dot-json packages/core/package.json version $npm_package_version
+
+echo "Please summarize this release in one sentence"
+read note
```

Comment:
The `read` command seem to interfere with `tee` command. Rearranging the command order is fixing this problem.